dynomitemanager-core/src/main/java/com/netflix/dynomitemanager/dynomite/DynomiteStandardTuner.java

```diff
@@ -117,6 +118,10 @@ public void writeAllProperties(String yamlLocation) throws IOException {
         entries.put("max_msgs", setMaxMsgs());
         entries.put("pem_key_file", floridaConfig.getDynomiteInstallDir() + "/conf/dynomite.pem");
 
+        if (!floridaConfig.getDynomiteHashtag().isEmpty()) {
+           entries.put("hashtag", floridaConfig.getDynomiteHashtag());
```

Comment:
can we verify that the hashtag is of length 2?

Reply:
done